kyaml/Makefile

```diff
@@ -39,6 +39,23 @@ vet:
 # likely related to https://github.com/kubernetes/kubernetes/issues/39188
 openapi:
 	sed -i 's/"x-kubernetes-patch-merge-key": "containerPort"/"x-kubernetes-patch-merge-key": "name"/g' 'openapi/kubernetesapi/swagger.json'
-	(which $(GOPATH)/bin/go-bindata || go get -v github.com/go-bindata/go-bindata)
+	(which $(GOPATH)/bin/go-bindata || go get -u github.com/go-bindata/go-bindata/...)
 	$(GOPATH)/bin/go-bindata --pkg kubernetesapi -o openapi/kubernetesapi/swagger.go openapi/kubernetesapi/swagger.json
 	$(GOPATH)/bin/go-bindata --pkg kustomizationapi -o openapi/kustomizationapi/swagger.go openapi/kustomizationapi/swagger.json
+
+kind:
+	which $(GOPATH)/bin/kind || GO111MODULE=on go get sigs.k8s.io/kind@v0.9.0
```

Comment:
The which isn't needed.  This should be a normal make target.
If the binary exists, the target shouldn't run.  The target should be
as in 
https://github.com/kubernetes-sigs/kustomize/blob/master/Makefile#L315

(define a MYGOBIN var up top, use it in the schema target)

Reply:
The way to test this is
```
make schema
rm $(go env GOPATH)/bin/kind
make schema
```